source-jira/connector_config.yaml

```diff
@@ -0,0 +1,32 @@
+api_token_sops: ENC[AES256_GCM,data:S2H7YakJ+VJlB10Q8zHAH79Lkm7tBGNE44AxZug0SlVidrq4QPQZTfvmFvDGGQ8Eaixr9peu3fuSn8gjbeckEoHHr8JiyLF8+WX7GQzpa3JvP+w3yMFbwoj/3WniaN/gdaB0ruPsPAwAswg9nvbYb1luEWLcnE+ksn4seIa3OROp7iBKwXal6n/OOoxNkDTC9fZjV6UHDr8mx6o5fIeHhBIBOv3LAaxeVKcbsuL1LNSz69AHaS8nBuYVNyAUVtPd,iv:Refwby8Rzy77ipyCM/Zf2rI0r46g6r1NIXSuJTOy56o=,tag:0zru+Trf/RDUtX9MfwOoXg==,type:str]
```

Comment:
I do not think these are the test credentials we want to use. Can we set up an estuary test account and use that instead?

Reply:
Actually, Dave gave me those credentials.

Reply:
Ah I see, would you mind double-checking with Dave that these were the credentials we should check in with the code and running as part of CI?